Source/Core/Core/NetPlayClient.cpp

```diff
@@ -824,14 +824,6 @@ bool NetPlayClient::StartGame(const std::string& path)
     // Needed to prevent locking up at boot if (when) the wiimotes connect out of order.
     NetWiimote nw;
```

Comment:
If I remove that line it fails to compile because the next line relies on it.  Should I delete both?